code/modules/mob/living/simple_animal/bot/medbot.dm

```diff
@@ -107,13 +97,15 @@ GLOBAL_VAR(medibot_unique_id_gen)
 /mob/living/simple_animal/bot/medbot/Initialize(mapload, new_skin)
 	. = ..()
 	skin = new_skin
+	synth_cooldown += world.time+600
 	update_icon()
 
 	var/datum/job/J = SSjob.GetJob(JOB_NAME_MEDICALDOCTOR)
 	access_card.access = J.get_access()
 	prev_access = access_card.access.Copy()
 
-	linked_techweb = SSresearch.science_tech
```

Comment:
If you could find a way to retain this and incorporate it into an "efficiency" increase for medbots, so they use less reagents than they're injecting that would be fantastic. It would also just be a good reason to use medbots if they *started* with an efficiency boost of some sort, even if you don't tie it to research. 

As an example of efficiency boost, here's a 20% efficiency boost:
Bot injects 10u, but only drains 8u from internal beaker/bag. I think this would be a good baseline efficiency without research, with it capping at somewhere from 40-60%

Reply:
done, caps at 65% with alien tech